src/sentry/api/serializers/models/group.py

```diff
@@ -572,3 +591,48 @@ def _get_seen_stats(self, item_list, user):
             }
 
         return attrs
+
+
+class StreamGroupSerializerSnuba(GroupSerializerSnuba, GroupStatsMixin):
+    def __init__(self, environment_ids=None, stats_period=None):
+        super(StreamGroupSerializerSnuba, self).__init__(environment_ids)
+
+        if stats_period is not None:
+            assert stats_period in self.STATS_PERIOD_CHOICES
+
+        self.stats_period = stats_period
+
+    def query_tsdb(self, group_ids, query_params):
+        if self.environment_ids is None:
```

Comment:
Is it correct to set all the stats to 0 if no environments are provided?

Reply:
fixed

Reply:
Nice, wasn't sure if intentional or not